src/DependencyLoader.php

```diff
@@ -19,23 +21,36 @@ public function __construct(DependencyParser $dependencyParser)
         $this->dependencyParser = $dependencyParser;
     }
 
+    /**
+     * @throws CommandExecutionException
+     */
     public function loadDependencies(string $composer, string $project): array
     {
         $commandOutput = $this->runComposerLicenseCommand($composer, $project);
 
-        return $this->dependencyParser->parse(join(PHP_EOL, $commandOutput));
+        return $this->dependencyParser->parse(implode("\n", $commandOutput));
```

Comment:
I think we should stay with the existing implementation here. I don't think this changes improves anything.